deployment/postgres-init-db/sql/tables/merlin/constraints/constraint_request.sql

```diff
@@ -0,0 +1,37 @@
+create table merlin.constraint_request (
+  id integer not null generated always as identity
+    primary key,
+  plan_id integer not null,
+  simulation_dataset_id integer not null, -- "not null", as we reject rqs if there aren't up-to-date sim results
+                                          -- (or if the provided sim dataset id is invalid)
+  force_rerun boolean,
+
+  -- Additional Metadata
+  requested_by text,
+  requested_at timestamptz not null default now(),
```

Comment:
Not sure if we typically comment these kinds of columns, but I noticed these ones are missing comments

Reply:
These are usually commented, thanks for catching this!

Reply:
Fixed!